portal-web/test/functional/com/liferay/portalweb/macros/Button.macro

```diff
@@ -25,10 +25,23 @@
 		<execute function="AssertClick" locator1="Button#DONE" value1="Done" />
 	</command>
 
+	<command name="clickEditIfPresent">
```

Comment:
hey @kyle-miho, this macro seems a bit over-protected. in essence, there is no case in which this macro can fail (which we want to happen should a failure condition happen). can you show me later what you're intending to do with this macro?